packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -147,6 +150,8 @@ app.all('*', authenticationRequired); // Require authentication for all routes
 
 ```
 
+See the following project that implements this code and provides a working example: [Okta Registration inline hook (with OAuth 2.0 authentication)](https://glitch.com/~okta-inlinehook-registrationhook-oauth2).
```

Comment:
I'm surprised that you don't use a global component for this snippet.

Reply:
Good point. I was awaiting the "include" component before single sourcing this type of content, and didn't want to add to the global components. But as there doesn't appear to be an "include" component on the horizon, let me give it a shot.